app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def is_alive(self) -> None:
+        if self.health <= 0:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
```

Comment:
I also searched for "how to split up a long f-string". Actually you can do it w/o that "\" symbol. Just use () after return. Looks much cleaner 
![Screenshot 2023-02-10 at 16 06 44](https://user-images.githubusercontent.com/122725491/218111671-b17a2cfc-beb2-4ffe-b62a-cbd891ab1652.png)

Reply:
@Anatolii-Poznyak, thanks! Looks really better! 
Lets wait for mentor comment about it.

Reply:
yeah, it is better to avoid " \ " symbol